templates/user/settings/profile.tmpl

```diff
@@ -12,14 +12,17 @@
 						<span class="text red tw-hidden" id="name-change-prompt"> {{ctx.Locale.Tr "settings.change_username_prompt"}}</span>
 						<span class="text red tw-hidden" id="name-change-redirect-prompt"> {{ctx.Locale.Tr "settings.change_username_redirect_prompt"}}</span>
 					</label>
-					<input id="username" name="name" value="{{.SignedUser.Name}}" data-name="{{.SignedUser.Name}}" autofocus required {{if or (not .SignedUser.IsLocal) .IsReverseProxy}}disabled{{end}} maxlength="40">
-					{{if or (not .SignedUser.IsLocal) .IsReverseProxy}}
+					<input id="username" name="name" value="{{.SignedUser.Name}}" data-name="{{.SignedUser.Name}}" autofocus required {{if or (or (not .SignedUser.IsLocal) ($.UserDisabledFeatures.Contains "change_username")) .IsReverseProxy}}disabled{{end}} maxlength="40">
```

Comment:
The same for `or` usage.

Reply:
Sorry for this mistake. Fixed in next commit